### PR TITLE
Remove G12 recovery code 🎉 

### PR DESCRIPTION
**Pull request by alex9smith**
https://trello.com/c/MJExPLo2/846-3-remove-g12-recovery-code-and-pii

Remove all the G12 recovery code :tada:

Reverts all G12 recovery PRs that haven't already been reverted:

#1347
#1341
#1335
#1334
#1333
#1331
#1330
#1329
#1325
#1318
#1317
#1314
#1310
#1303
#1301
#1299
#1298
#1297
#1295
#1292
#1288

There's a lot. Sorry to whoever is reviewing this.

### CODE REVIEW

**Comment by lfdebrux** [comment]
### My review checklist

I've:

- [ ] looked at the code
- [ ] tested locally
- [ ] run functional tests locally

File: config.py [hunk 1]
```diff
@@ -122,7 +119,7 @@ class Test(Config):
     SHARED_EMAIL_KEY = "KEY"
 
     DM_MAILCHIMP_USERNAME = 'not_a_real_username'
-    DM_MAILCHIMP_API_KEY = 'not_a_real_key'  # pragma: allowlist secret
+    DM_MAILCHIMP_API_KEY = 'not_a_real_key'
```
Comment (lfdebrux) [inline config.py:122]: We might need to keep this pragma?

**Comment by lfdebrux** [comment]
I think you missed https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/1305 (changes from the PR are still present in this branch)

File: config.py (hunk 1 above)
Comment (alex9smith) [reply config.py:122]: Ah yes. Am I missing something that generates / checks for these?

Comment (lfdebrux) [reply config.py:122]: Have you got https://github.com/alphagov/gds-pre-commit set up on your machine?

Comment (alex9smith) [reply config.py:122]: No! That'd do it

**Review by lfdebrux — APPROVED** [review]
This looks good to me, merge away 🚢

**Pull request merged**